docs/src/pages/components/view/demo.tsx

```diff
@@ -1,286 +1,82 @@
-            type="text"
-            placeholder="Set maxHeight"
-            value={maxHeight}
-            onChange={(event: any) => {
-              setMaxHeight(event.target.value);
-            }}
-          />
-        </FieldSet>
-
-        <FieldSet id="maxWidth">
-          <input
-            type="text"
-            placeholder="Set maxWidth"
-            value={maxWidth}
-            onChange={(event: any) => {
-              setMaxWidth(event.target.value);
-            }}
-          />
-        </FieldSet>
-
-        <FieldSet id="minHeight">
-          <input
-            type="text"
-            placeholder="Set minHeight"
-            value={minHeight}
-            onChange={(event: any) => {
-              setMinHeight(event.target.value);
-            }}
-          />
-        </FieldSet>
-
-        <FieldSet id="minWidth">
-          <input
-            type="text"
-            placeholder="Set minWidth"
-            value={minWidth}
-            onChange={(event: any) => {
-              setMinWidth(event.target.value);
-            }}
-          />
-        </FieldSet>
-
-        <FieldSet id="role">
-          <input
-            type="text"
-            placeholder="Set role"
-            value={role}
-            onChange={(event: any) => {
-              setAriaRole(event.target.value);
-            }}
-          />
-        </FieldSet>
-
-        <FieldSet id="opacity">
-          <input
-            type="text"
-            placeholder="Set opacity"
-            value={opacity}
-            onChange={(event: any) => {
-              setOpacity(event.target.value);
-            }}
-          />
-        </FieldSet>
+    `<View` +
+    (props.ariaLabel ? `\n  ariaLabel="${props.ariaLabel}"` : '') +
+    `\n  as="${props.as}"` +
+    (props.backgroundColor
+      ? `\n  backgroundColor="${props.backgroundColor}"`
+      : '') +
+    (props.borderRadius ? `\n  borderRadius="${props.borderRadius}"` : '') +
+    (props.border ? `\n  border="${props.border}"` : '') +
+    (props.boxShadow ? `\n  boxShadow="${props.boxShadow}"` : '') +
+    (props.color ? `\n  color="${props.color}"` : '') +
+    (props.height ? `\n  height="${props.height}"` : '') +
+    (props.opacity ? `\n  opacity="${props.opacity}"` : '') +
+    (props.padding ? `\n  padding="${props.padding}"` : '') +
+    (props.width ? `\n  width="${props.width}"` : '') +
+    `\n  onClick={() => alert('🏔 What a beautiful <View>! 🔭')}
+  >
+  {"I\'m a <${props.as}>! 🤩"}
+</View>`
+  );
+};
 
-        <FieldSet id="padding">
-          <input
-            type="text"
-            placeholder="Set padding"
-            value={padding}
-            onChange={(event: any) => {
-              setPadding(event.target.value);
-            }}
-          />
-        </FieldSet>
+export const ViewDemo = () => {
+  const props = useViewProps({
+    as: 'div',
+    ariaLabel: 'View example',
+    width: '20rem',
+    height: '4rem',
+    color: 'var(--amplify-colors-blue-60)',
```

Comment:
It's an interesting idea to use CSS variables in the example. Do you think this will be intuitive to customers though?

Reply:
I'm also on the fence, what does everyone else think? I liked it because it exposed the idea to customers that they could just use the CSS variables. But maybe there could be a better user experience for exposing the CSS variables, like a searchable dropdown instead?

Reply:
:shipit: